backend/coins/eth/account.go

```diff
@@ -208,41 +208,12 @@ func (account *Account) Initialize() error {
 	)
 
 	account.coin.Initialize()
-	done := account.Synchronizer.IncRequestsCounter()
-	go account.poll(done)
+	account.initDone = account.Synchronizer.IncRequestsCounter()
+	account.EnqueueUpdate()
```

Comment:
`go account.EnqeueuUpdate()` to not block here in case the channel is full (considering my other comment where EnqueueUpdate() should drop the default case)